docs/_docs/configuration/options.md

```diff
@@ -44,8 +44,9 @@ class="flag">flags</code> (specified on the command-line) that control them.
       <td>
         <p class="name"><strong>Safe</strong></p>
         <p class="description">
-          Disable <a href="/docs/plugins/">custom plugins</a>, caching to disk
-          and ignore symbolic links.
+          Disable <a href="/docs/plugins/">plugins</a>, caching to disk
+          and ignore symbolic links.  </br>
+          Whitelisted plugins are not disabled.
```

Comment:
How about rephrasing to the following?
```suggestion
          Disable <a href="/docs/plugins/">non-whitelisted plugins</a>, caching to disk
          and ignore symbolic links.
```

Reply:
I like it. The only reason I had it separate was to link "whitelisted" to the config page for whitelisting.

Reply:
I don't think we have a dedicated section on the `whitelist` configuration..
If that's indeed the case, I think you can add a *note* to the page at `/docs/plugins/` informing the visitor about the whitelist..
(*it can be part of this PR itself*)